### DISCUSSION TITLE
i want code

### DISCUSSION THREAD

**Discussion by hariommahato** [shakilkhan12/chawkbazar-eCommerce#2] [General]
please provide me code

**Comment by shakilkhan12** [comment]
Brother Just click on the download button what do you want more?

**Comment by bishalbhowmik** [comment]
Server and client side  is running. But the data not loaded. Spinner is loading in the client side and Server side this message is showing: The `uri` parameter to `openUri()` must be a string, got "undefined". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.up .
How to fix this problem?

**Reply by shakilkhan12** [reply]
Please show me your .env file and your connection file
